channeldb/meta.go

```diff
@@ -177,3 +187,61 @@ func (d *DB) putOptionalMeta(om *OptionalMeta) error {
 		return metaBucket.Put(optionalVersionKey, b.Bytes())
 	}, func() {})
 }
+
+// CheckMarkerPresent returns the marker under the requested key or
+// ErrMarkerNotFound if either the root bucket or the marker key within that
+// bucket does not exist.
+func CheckMarkerPresent(tx kvdb.RTx, markerKey []byte) ([]byte, error) {
```

Comment:
Newly added functions are missing unit tests.

Reply:
Fair point! Added unit tests for all new or newly exported functions.